.llvm-ci.yml

```diff
@@ -82,3 +82,73 @@ jobs:
       packagePublishDescription: 'Verona LLVM $(BuildType) Build for Linux at $(GOOD_HASH)'
       versionOption: custom
       versionPublish: $(VERSION).$(Build.Buildid)
+
+- job:
+  displayName: LLVM Windows Build
+  pool: verona-llvm-build-windows
+  timeoutInMinutes: 120
+  strategy:
+    matrix:
+      Debug:
+        BuildType: Debug
+        BuildName: debug
+      Release:
+        BuildType: Release
+        BuildName: release
+  # TODO: Figure how to set version on Windows
```

Comment:
What's the issue? Can you not do the same as in Linux?

Reply:
I can install `grep` with choco, but `coreutils` (for `cut`) doesn't seem to be available. I just need to figure out but that's not important because none of those builds will be off a release branch anyway, sothe LLVM version doens't matter much.

Reply:
Can you  use `wsl` for somethings?

Reply:
I probably could, but not before I get the MLIR generator upstream. For now, anything reasonable that works is better than trying inprovements.

This build is a step towards getting my changes upstream, so they can be tested and we can continue the development in multiple layers (ast, mlir, rt).